tox.ini

```diff
@@ -27,8 +27,8 @@ commands =
     unittest: coverage xml
     pytest: pytest --cov=. --ignore=.tox --disable-pytest-warnings --cov-report=xml --cov-append {toxinidir}
 setenv =
-    PYTHONDONTWRITEBYTECODE=1
```

Comment:
may be leave PYTHONDONTWRITEBYTECODE=1 ?

Reply:
What this give us? As far I know the .pyc files will be ignored if the versions don't match.

Reply:
Ok. Fixed